### ISSUE TITLE
About Yolov3

### ISSUE THREAD

**Issue by HigashikataZhangsuke** [diggerdata/kolt_ros#1]
This is really a good work. I want to know may yolov3 or tiny yolo be used on the framework to substitute yolov2?Thanks!

**Comment by germal** [comment]
Hi @sjtuzyz , did you managed to make work kolt with yolov2 ? Which tensorflow/keras versions did you use ?
Thanks

**Comment by HigashikataZhangsuke** [comment]
> Hi @sjtuzyz , did you managed to make work kolt with yolov2 ? Which tensorflow/keras versions did you use ?
> Thanks

My project due was June, So I actually didn't try to use this Repo because the author didn't reply to me.I finally did my job without using ROS.Sorry for not providing you with available help.

**Comment by germal** [comment]
Thank you for your reply ! DId you implemented a similar project with yolov3 and Realsense ?

**Comment by HigashikataZhangsuke** [comment]
> Thank you for your reply ! DId you implemented a similar project with yolov3 and Realsense ?

No,I just use a simple cellphone camera to get data,And I didn't use ROS. So I think maybe my project isn't a good enough choice for you because it is so poor.But anyway if you want to have a look. https://github.com/sjtuzyz/Tracking_by_detection

**Comment by germal** [comment]
Actually I think that it is very well done and well documented as well

Thank you !
